.travis.yml

```diff
@@ -1,7 +1,6 @@
 language: go
 
 go:
-  - 1.8.x
-  - 1.9.x
+  - 1.9
```

Comment:
WDYT about `1.9.x` ? or `1.x` (for the most recent version)?

Reply:
Good point. I reverted to `1.9.x` and added master to be future proof.